test/MinImports.lean

```diff
@@ -75,8 +75,13 @@ import Mathlib.Data.Nat.Notation
 lemma hi (n : ℕ) : n = n := by extract_goal; rfl
 
 /--
-warning: Imports increased to
+warning: Imports increased by 398 to
 [Init.Guard, Mathlib.Data.Int.Notation]
+
+Now redundant: []
```

Comment:
Maybe only print this if it is non-empty?

Reply:
Only printing when non-empty.